vmm/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 default = []
 dbus_api = ["blocking", "futures", "zbus"]
 guest_debug = ["kvm", "gdbstub", "gdbstub_arch"]
-igvm = []
+igvm = ["igvm_parser", "igvm_defs"]
```

Comment:
As what I commented earlier, this is where `"igvm_parser", "igvm_defs"` are being used the first time, and should be the where these dependencies are introduced.